frontend/src/routes/index.tsx

```diff
@@ -1,35 +1,52 @@
-import { createFileRoute } from '@tanstack/react-router';
-import { PageLayout } from '@layouts';
-import { Button } from '@components';
+import { createFileRoute, Link } from '@tanstack/react-router';
 
 export const Route = createFileRoute('/')({
   component: App,
 });
 
+const landingButtonData = [
+  {
+    subtitle: '보호자 서비스',
+    title: '병원 동행 신청하기',
+    image: '/images/hospital.png',
+    link: '/customer',
+  },
+  {
+    subtitle: '도우미 서비스',
+    title: '일감 찾기',
+    image: '/images/work.png',
+    link: '/helper',
+  },
+];
+
 function App() {
   return (
-    <PageLayout>
-      <PageLayout.Header title='토닥' showBack={true} showClose={false} background={true} />
-      <PageLayout.Content>
-        <div className='h-full text-center'>
-          <header className='body1-16-medium bg-neutral-0 flex h-full flex-col items-center justify-center text-neutral-100'>
-            <p>토닥 서비스 준비중입니다.</p>
-            <a
-              className='text-[#61dafb] hover:underline'
-              href='https://github.com/softeerbootcamp-6th/Team4-PopoPony'
-              target='_blank'
-              rel='noopener noreferrer'>
-              팀 레포지토리 바로가기
-            </a>
-          </header>
-        </div>
-        <div className='h-[100dvh]'></div>
-      </PageLayout.Content>
-      <PageLayout.Footer>
-        <Button variant='primary' onClick={() => alert('Single 버튼 클릭됨')}>
-          확인
-        </Button>
-      </PageLayout.Footer>
-    </PageLayout>
+    <div className='h-full bg-[url("/images/landing-background.png")] bg-cover bg-center px-[2rem]'>
+      <div className='flex-col-center gap-[1.6rem] pt-[4.8rem]'>
+        <img src='/images/logo-text.svg' alt='logo-text' className='w-[7.2rem]' />
+        <h2 className='title-20-bold text-text-neutral-secondary'>토닥과 함께, 투-닥터</h2>
```

Comment:
피그마 원본에는 느낌표가 있는 것 같습니다!

```suggestion
        <h2 className='title-20-bold text-text-neutral-secondary'>토닥과 함께, 투-닥터!</h2>
```